include/crm/cib/cib_types.h

```diff
@@ -67,6 +67,13 @@ enum cib_call_options {
     cib_force_diff      = 0x10000000
 };
 
+enum cib_change_section_info {
+    cib_change_section_none     = 0x00000000,
+    cib_change_section_nodes    = 0x00000001,
+    cib_change_section_alerts   = 0x00000002,
+    cib_change_section_status   = 0x00000004
+};
+
```

Comment:
I'm thinking this should be in cib/internal.h for now. External code could benefit from it, but I'd rather get some experience with it before exposing it, in case anything unexpected requires changes.

Reply:
Okay!
Thanks.